COORDINATE_SYSTEM2/COORDINATE_SYSTEM2/CoordinateSystem.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace COORDINATE_SYSTEM2
+{
+    internal class CoordinateSystem
+    {
+        public List<Point> listOfPoints = new List<Point>(); //List with all Points
+
+        public List<double> listOfDistsPoints = new List<double>(); //List with all Distances between the points
+
+        public void Add(Point point)
+        {
+            listOfPoints.Add(point);
+        }
+
+        public void DisplayAllPoints()
+        {
+            int count = 1;
+
+            foreach (Point i in listOfPoints)
+            {
+                Console.Write($"point {count++} : {i};\n");
+            }
```

Comment:
If you need index, it's better to use 'for' loop

```suggestion
            for (var i = 0; i < listOfPoints.Count; ++i)
            {
                Console.Write($"point {i} : {listOfPoints[i]};\n");
            }
```

Reply:
done